client/my-sites/stats/stats-redirect/index.tsx

```diff
@@ -89,13 +98,10 @@ const StatsRedirectFlow: React.FC< StatsRedirectFlowProps > = ( { children } ) =
 		);
 
 		return <></>;
-	} else if ( ! isFetching ) {
+	} else if ( ! isFetching || ( canUserViewStats && ! canUserManageOptions ) ) {
```

Comment:
Can you please explain more about the condition here?

Reply:
For users other than admins, fetching status doesn't matter, and I display the page anyway.

Reply:
Just curious, would they fall into the condition on L80? 🤔

Reply:
No, because `purchaseNotPosponed` would be false for them due to a failed fetch.

Reply:
In this case, those users without admin access would never see the StatsLoader even when it is loading. Would it be a potential issue?

Reply:
I don't think the condition would happen because fetching is now disabled for users without `manage` permissions.